requests/cookies.py

```diff
@@ -120,8 +120,7 @@ def extract_cookies_to_jar(jar, request, response):
     :param request: our own requests.Request object
     :param response: urllib3.HTTPResponse object
     """
-    if not (hasattr(response, '_original_response') and
-            response._original_response):
+    if getattr(response, '_original_response', None) is None:
```

Comment:
This one doesn't need to be an `is None`, sorry: given that we're doing a bool test, it just needs to fall back to `None`.